website/docs/r/resource_tag.html.markdown

```diff
@@ -45,7 +45,7 @@ Review the argument references that you can specify for your resource.
 
 - `resource_id` - (Required, String) The CRN of the resource on which the tags is be attached.
 - `resource_type` - (Optional, String) The resource type on which the tags should be attached.
-- `tag_type` - (Optional, String) Type of the tag. Supported values are: `user`, `service`, or `access`. The default value is user.
```

Comment:
Why was `service`tag type removed

Reply:
We decided to remove the `service` tag due to the fact that the `service` tag is used in IBM internally only and only a little set of users can use it. So it should not be exposed in the documentation.